script.js

```diff
@@ -0,0 +1,432 @@
+        fillSpell();
+    });
+    //function when the background drop down is clicked
+    $('#backDrop').on("click", function (event) {
+        a = event.target.innerText;
+        $('.backbtn').text(a)
+    });
+    //function when the level drop down is clicked    
+    $('#levelDrop').on("click", function (event) {
+        a = event.target.innerText;
+        $('.levelbtn').text(a)
+    });
+    //function to fill the class using api get method    
+    function fillClass() {
+        const pathUrl = "https://api.open5e.com/classes/";
+        $.ajax({
+            url: pathUrl,
+            type: "GET",
+            dataType: "json",
+            success: function (result) {
+                for (var i = 0; i < result.results.length; i++) {
+                    var li = "<li>" + result.results[i].name + "</li>";
+                    $("#classDrop").append(li);
+                }
+            },
+            error: function (result) {
+
+            }
+        });
+    }
+    //Function to roll a D20 using diceapi
+    $('#click-to-roll').on("click", function (event) {
+        var diceURL = "https://cors-anywhere.herokuapp.com/http://roll.diceapi.com/json/d20/"
+        var rollValue = -1
+        $.ajax({
+            url: diceURL,
+            type: "GET",
+            dataType: "json",
+            success: function (result) {
+                rollValue = result.dice[0].value
+                console.log(rollValue)
+                var resultModal = document.getElementById('diceResult')
+                resultModal.innerHTML = "You rolled a " + rollValue
+            },
+            error: function (result) {
+            }
+        });
+        $('.modal').modal();
+    });
+
+    //function to fill the race using api get method    
+    var raceUrl = "https://api.open5e.com/races/";
+    $.ajax({
+        url: raceUrl,
+        type: "GET",
+        dataType: "json",
+        success: function (result) {
+            for (var i = 0; i < result.results.length; i++) {
+                var li = "<li>" + result.results[i].name + "</li>";
+                $("#raceDrop").append(li);
+            }
+        },
+        error: function (result) {
+
+        }
+    });
+
+    function fillSpell() {
+        var p = "";
+        var check = true;
+        for (var page = 1; page <= 7; page++) {
+            const spellUrl = "https://api.open5e.com/spells/?page=" + page;
+            $.ajax({
+                url: spellUrl,
+                type: "GET",
+                dataType: "json",
+                success: function (result) {
+                    for (var i = 0; i < result.results.length; i++) {
+                        var arrayCheck = [];
+                        arrayCheck = result.results[i].dnd_class.split(",");
+                        for (var j = 0; j < arrayCheck.length; j++) {
+
+                            if (arrayCheck[j] == classCheck) {
+                                if (check == true) {
```

Comment:
you should try to make these truthy checks, these can be simplified to

```
if (check) {
```

Unless you are really concerned that it has to be a boolean value, it's a best practice to keep it short and sweet